README.md

```diff
@@ -28,240 +26,245 @@ pip install -r requirements.txt
-
-The [corresponding script](source/4_jungfraujoch_active_learning.py) does this in 7 steps:
-1. It defines an Active Learning agent connecting to the LightlyAPI. 
-   This agent needs to know your API token and the dataset id, which need to be provided as environment variables.
-2. It reads the label file from LabelStudio with a [helper function](source/read_LabelStudio_label_file.py).
-3. It defines an image classification model as a pytorch CNN. It uses a resnet18 as backbone.
-    If you want to learn more about the model and/or change it to another model, look at the [classification_model.py](source/classification_model.py).
-4. It trains the model for a few epochs on the 30 labeled images.
-5. It predicts with the model on the complete dataset. 
-   The predictions are stored in a lightly `ClassificationScorer` to calculate active learning scores.
-    The scores measure how uncertain the model is in classifying a certain sample.
-6. Another sampling is performed, sampling 15 images until we have 45 images.
-   It uses the CORAL sampler, which combines CORESET and active learning to choose samples 
-   which have both a high prediction uncertainty 
-   and are different to each other and already chosen samples.
-7. The newly chosen samples need to be copied to the directory with the labeled images again. 
-    Thus the corresponding `lightly-download` command is defined.
+Rename the file to `annotation-0.json` and place that in the root directory of this repository.
 
-```bash
-# WEATHER_DIR_RAW and WEATHER_DIR_LABELED must already by set.
-# This script uses the label file named `weather_labels_iter0_30.json`
-# We also need to set the environment variables for the Lightly Webapp connection.
-# They are the same as the ones used in the lightly-download command at the end of step 2.
-export LIGHTLY_TOKEN=MY_TOKEN
-export LIGHTLY_DATASET_ID_WEATHER=DATASET_ID
-python source/4_jungfraujoch_active_learning.py
+## 4. Train a model and prepare for active learning
+
+We can train a classification model with the 30 labelled samples. Script [train_model_1.py](./source/train_model_1.py) loads samples from `annotation-0.json` and performs this task.
+
+```sh
+python source/train_model_1.py
 ```
 
-With the just printed CLI command, you can download the filenames of the newly chosen images
-and copy them to the directory with the labeled images. Execute this CLI command.
+The following steps are performed in this script:
+* Load the annotations and the labelled images.
+* Load the validation set.
+* Train a simple model as in [model.py](./source/model.py).
+* Make predictions for all samples for training, including unlabelled samples.
+* Dump the predictions in [Lightly Prediction format](https://docs.lightly.ai/docs/prediction-format#prediction-format) into directory `lightly_predictions`.
 
+We can see that the model performance is not good:
+```
+Training Acc:  60.000           Validation Acc:  19.027
```

Comment:
I know. Overfitting is expected in this case because the model is trained on 30 samples only. The main idea here is that the model performs poorly on the validation set.